bin/demeuk.py

```diff
@@ -1145,11 +1155,11 @@ def main():
         'skip': False,
 
         # Modify
-        'encode': True,
-        'mojibake': True,
-        'tab': True,
-        'trim': True,
-        'newline': True,
+        'encode': False,
```

Comment:
Agreed, personally docopt was an intersting idea, but I guess it would makes senso to port demeuk to a different args parser and rewrite this code. I'll create an issue for that.

Reply:
Thus, not implementing it for now.